autoscale/src/main/java/com/sequenceiq/periscope/monitor/context/EvaluatorContext.java

```diff
@@ -3,4 +3,6 @@
 public interface EvaluatorContext {
 
     Object getData();
+
+    long getClusterOrStackId();
```

Comment:
Maybe `getItemId` would be a better name

Reply:
renamed to getItemId